### PR TITLE
[3.0.0] Fix REST API issues

### PR DESCRIPTION
**Pull request by Sarangan0219**
## Purpose

Fixes https://github.com/wso2/docs-apim/issues/1987

## Affected products

APIM 3.0.0

### CODE REVIEW

File: en/docs/develop/product-apis/getting-started/guide-admin-v0.15.md [hunk 1]
```diff
@@ -59,3 +59,8 @@ Token response:
 ```
 
 Now, you have a valid access token, which you can use to invoke an API. Navigate through the API descriptions to find the required API, obtain an access token as described above and invoke the API with the authentication header. If you use a different authentication mechanism, this process may change.
+
+Further, to configure the Admin Portal REST API please do the necessary configurations mentioned in [configuring REST APIs]({{base_path}}/develop/product-apis/configuring-rest-api/).
```
Comment (tgtshanika) [inline en/docs/develop/product-apis/getting-started/guide-admin-v0.15.md:63]: Avoid using Please

File: en/docs/develop/product-apis/getting-started/guide-admin-v0.15.md [hunk 2]
```diff
@@ -59,3 +59,8 @@ Token response:
...
+
+!!! note
+    To get a detailed overview of Admin Portal REST API follow [Admin API v0.15]({{base_path}}/develop/product-apis/admin-apis/admin-v0.15/admin-v0.15/).
```
Comment (tgtshanika) [inline en/docs/develop/product-apis/getting-started/guide-admin-v0.15.md:66]: API, follow

File: en/docs/develop/product-apis/getting-started/guide-admin-v0.15.md (hunk 1 above)
Comment (Sarangan0219) [reply en/docs/develop/product-apis/getting-started/guide-admin-v0.15.md]: https://github.com/wso2/docs-apim/pull/2554/commits/8e869593dda7ae51f6e26b3a5c4a9ecc4f7fd2b1

File: en/docs/develop/product-apis/getting-started/guide-admin-v0.15.md (hunk 2 above)
Comment (Sarangan0219) [reply en/docs/develop/product-apis/getting-started/guide-admin-v0.15.md]: https://github.com/wso2/docs-apim/pull/2554/commits/8e869593dda7ae51f6e26b3a5c4a9ecc4f7fd2b1

**Review by jaadds — APPROVED** [review]

**Pull request merged**